crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -441,6 +496,7 @@ impl LogletConfiguration {
             LogletConfiguration::Replicated(configuration) => {
                 build_new_replicated_loglet_configuration(
                     configuration.loglet_id.next(),
+                    &metadata().nodes_config_snapshot(),
```

Comment:
Why not `nodes_config_ref`?

Reply:
Frankly, because until very recently, I didn't understand the distinction! ;-) This is indeed a better choice here, thank you!